erica/erica_legacy/pyeric/eric_errors.py

```diff
@@ -263,6 +264,94 @@
                         **_ERIC_GLOBAL_INITIALISATION_ERRORS, **_ERIC_GLOBAL_ERRORS, **_ERIC_TRANSFER_ERRORS,
                         **_ERIC_CRYPT_ERRORS, **_ERIC_IO_ERRORS, **_ERIC_PRINT_ERRORS}
 
+_ERIC_ERRORS_NO_RETRY = ["ALREADY_OPEN_UNLOCK_CODE_REQUEST", "ELSTER_REQUEST_ID_UNKNOWN", "ERIC_GLOBAL_PRUEF_FEHLER",
+                         "ERIC_GLOBAL_LANDESNUMMER_UNBEKANNT",
+                         "ERIC_GLOBAL_BUFANR_UNBEKANNT", "ERIC_GLOBAL_LANDESNUMMER_BUFANR",
+                         "ERIC_GLOBAL_LANDESNUMMER_BUFANR", "NO_MATCHING_IDENTIFIER_FOR_UNLOCK_REQUEST",
```

Comment:
Could you also delete "NO_MATCHING_IDENTIFIER_FOR_UNLOCK_REQUEST" as an error altogether? We don't use it ever and Julian and I couldn't even reconstruct what this was ever used for. :x

Reply:
Done!

Reply:
Oh sorry! I will delete it! What about the counter 1-7 of the ERIC_CUSTOM_ERROR_CODES list? Should I reorganize so that we still have the 2 or I just remove that error and we have the 2 missing? I suppose those are the codes we get from ELSTER?

Reply:
No, the way you did it is correct, I think. We heavily rely on those error codes in the webapp and only check for the code as far as I remember. And the ones in this list actually are custom errors that *we* introduced that are special cases of more general Elster Errors.

Reply:
Alrighty! Thanks :)